packages/network_info_plus/network_info_plus/ios/Classes/getgateway.c

```diff
@@ -13,8 +13,14 @@
 #include <sys/sysctl.h>
 #include "getgateway.h"
 
-#include "route.h"
-#define TypeEN    "en0"
+#include "TargetConditionals.h"
+#if TARGET_IPHONE_SIMULATOR
+    #include <net/route.h>
+    #define TypeEN    "en1"
+#else
+    #include "route.h"
+    #define TypeEN    "en0"
+#endif
```

Comment:
This seems not to be in a correct PR. if you want to fix an issue, it would be nice if you can send another PR for this and mention the issue for it.